src/components/JumpstartAccordion.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+} from "@material-ui/core";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+interface JumpstartAccordionProps {
+  readonly sections: Array<{
+    readonly body: string;
```

Comment:
This is good I should probably do this for the other prop interfaces I made before

Reply:
Which part?

Usually I actually prefer having separate object interfaces, so like `readonly sections: Section[]`

Reply:
readonly, I dont know much about Typescript but it seems like a good idea.

Reply:
Ah yeah it's a good practice generally to prevent mutating props, TS just makes it easy. I'm new to TS but a big fan so far.